spec/basic/expected_conditions_spec.js

```diff
@@ -0,0 +1,167 @@
+var EC = protractor.ExpectedConditions;
+
+describe('expected conditions', function() {
+  beforeEach(function() {
+    browser.get('index.html#/form');
+  });
+
+  it('should have alertIsPresent', function() {
+    var alertIsPresent = EC.alertIsPresent();
+    expect(alertIsPresent.call()).toBe(false);
```

Comment:
I might be being dense, but why not:

`expect(alertIsPresent()).toBe(false);`?